framework/src/modules/network/index.js

```diff
@@ -0,0 +1,60 @@
+const defaults = require('./defaults');
+const Network = require('./network');
+const BaseModule = require('../base_module');
+
+/* eslint-disable class-methods-use-this */
+
+/**
+ * Network module specification
+ *
+ * @namespace Framework.Modules
+ * @type {module.NetworkModule}
+ */
+module.exports = class NetworkModule extends BaseModule {
+	/* eslint-disable-next-line no-useless-constructor */
+	constructor(options) {
+		super(options);
+	}
+
+	static get alias() {
+		return 'network';
+	}
+
+	static get info() {
+		return {
+			author: 'LiskHQ',
+			version: '0.1.0',
+			name: 'lisk-core-network',
+		};
+	}
+
+	get defaults() {
+		return defaults;
+	}
+
+	get events() {
+		return [];
+	}
+
+	get actions() {
+		return {
+			request: async action => this.network.actions.request(action),
```

Comment:
getNodeInfo need to be added as actions too? Maybe needed to get the height and also, applyNodeInfo in order to update the height? or maybe applyNodeInfo is should not be updated but network should listen to new block event to update its height?

Reply:
I didn't expose it on purpose because P2PNodeInfo is currently the same as system peer headers and it's better to keep it in one place.